backend/docs/api/inventory.openapi.yaml

```diff
@@ -0,0 +1,1752 @@
+openapi: 3.0.1
+info:
+  title: Device inventory
+  description: >
+    An API for uploading device attributes.
+
+
+    Devices can upload vendor-specific attributes (software/hardware info,
+    health checks, metrics, etc.) of various data types to the backend.
+  version: '1'
+
+tags:
+  - name: Inventory Device
+  - name: Inventory Internal
+  - name: Inventory Internal v2
+  - name: Inventory Management
+  - name: Inventory Management v2
```

Comment:
as a part of the review I counted the v2 management endpoints (while checking if all have tags) there are only 2 v2 management endpoints. just to double check, correct, right? and only one internal v2.